resources/schema.gql

```diff
@@ -3065,28 +3107,205 @@ type ModifyPasswordWrongOldPassword implements ModifyPasswordResult {
 
 type Molecule {
 	"""
-	Looks up the project
+	Looks up the project.
+	
+	Shortcut for `v1 { project }`.
 	"""
-	project(ipnftUid: String!): MoleculeProject
+	project(ipnftUid: String!): MoleculeProject @deprecated(reason: "Use `v2 { project }` instead.")
 	"""
-	List the registered projects
+	List the registered projects.
+	
+	Shortcut for `v1 { projects }`.
 	"""
-	projects(page: Int, perPage: Int): MoleculeProjectConnection!
+	projects(page: Int, perPage: Int): MoleculeProjectConnection! @deprecated(reason: "Use `v2 { projects }` instead.")
 	"""
 	Latest activity events across all projects in reverse chronological
-	order
+	order.
+	
+	Shortcut for `v1 { activity }`.
 	"""
-	activity(page: Int, perPage: Int): MoleculeProjectEventConnection!
+	activity(page: Int, perPage: Int): MoleculeProjectEventConnection! @deprecated(reason: "Use `v2 { activity }` instead.")
+	"""
+	1-st Molecule API version (query).
+	"""
+	v1: MoleculeV1! @deprecated(reason: "Use `v2` instead")
+	"""
+	2-nd Molecule API version (query).
+	"""
+	v2: MoleculeV2!
+}
+
+input MoleculeActivityFiltersV2 {
+	"""
+	This filter is provided as an example.
+	"""
+	byIpnftUids: [String!]
+}
+
+type MoleculeAnnouncementEntryV2 {
+	project: MoleculeProjectV2!
+	id: String!
+	headline: String!
+	body: String!
+	attachments: [String!]!
+	accessLevel: String!
+	changeBy: AccountID!
+	categories: [String!]!
+	tags: [String!]!
+}
+
+type MoleculeAnnouncementsDatasetMutV2 {
+	"""
+	Creates an announcement record for the project.
+	"""
+	create(		headline: String!,		body: String!,
+		"""
+		List of dataset DIDs to link
+		"""
+		attachments: [String!],		moleculeAccessLevel: String!,		moleculeChangeBy: String!,		categories: [String!]!,		tags: [String!]!
+	): CreateAnnouncementResult!
+}
+
+type MoleculeAnnouncementsDatasetV2 {
+	"""
+	Access the underlying core Dataset
+	"""
+	dataset: Dataset!
+	byId(id: String!): MoleculeAnnouncementEntryV2!
+}
+
+type MoleculeDataRoomDatasetV2 {
+	"""
+	Access the underlying core Dataset
+	"""
+	dataset: Dataset!
+	entries(pathPrefix: CollectionPath, maxDepth: Int, page: Int, perPage: Int): MoleculeDataRoomEntryV2Connection!
```

Comment:
Data room entries listing should also have `filters` parameter to enable filtering by `tags`, `categories`.

Additionally I think we should include `accessLevel` into the filters too. Currently Molecule fetches all entries and filters them on their end based on what access level user has, but this doesn't work well with pagination. I think instead they should be telling us what access levels are allowable and we will account for those in filtering on our side.